src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -177,6 +177,7 @@ public BasePanel(JabRefFrame frame, BasePanelPreferences preferences, BibDatabas
         setupMainPanel();
 
         setupActions();
+        setPreviewOn(true);
```

Comment:
Now the preview is always shown?

Reply:
Yes at the beginning only the preference is set to be true but it's not shown until a row is pressed. As per previous logic, the preview entry in menu is ticked when the program loads but it does not show any preview even a row is clicked.

Reply:
@tobiasdiez Any specific ideas about this?